protostar/commands/init/project_creator/_project_creator.py

```diff
@@ -31,28 +31,28 @@ def copy_template(self, cairo_version: CairoVersion, project_root_path: Path):
                 f"Folder or file named {project_root_path.name} already exists. Choose different project name."
             ) from ex_file_exists
 
-    def save_protostar_toml(
-        self, project_root_path: Path, cairo_version: CairoVersion
+    def save_protostar_toml(self, project_root_path: Path) -> None:
+        config = ConfigurationFileV2Model(
+            protostar_version=str(self._protostar_version),
+            contract_name_to_path_strs={"main": ["src/main.cairo"]},
+            project_config={
+                "lib-path": "lib",
+                "linked-libraries": ["src"],
+            },
+            command_name_to_config={},
+            profile_name_to_project_config={},
+            profile_name_to_commands_config={},
+        )
```

Comment:
This is a really weird place to put this config creation. In which scenarios is it saved? Adapted projects only?